internal/pkg/test/cache.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package test
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+// SetCacheDir creates a new image cache in the context of testing
+// and set the appropriate environment variable to ensure that the
+// temporary cache is used by the caller. Note that the context of
+// the directory that is passed in will be deleted when calling
+// CleanCacheDir().
```

Comment:
Think this comment needs updating, something like:

```
// SetCacheDir creates a new temporary cache directory in the directory basedir
// and returns the path of the new directory. If basedir is the empty string,
// SetCacheDir uses the default directory for temporary files (see os.TempDir).
// It is the caller's responsibility to remove the directory when no longer
// needed, which can be accomplished by calling CleanCacheDir.
```

Reply:
Agreed